commands/audit/audit_test.go

```diff
@@ -933,3 +933,125 @@ func TestCreateResultsContext(t *testing.T) {
 		}
 	}
 }
+
+func TestAudit_DiffScanFlow(t *testing.T) {
+	testDirPath := filepath.Join("..", "..", "tests", "testdata", "projects", "jas", "jas")
+	tempDirPath, createTempDirCallback := coreTests.CreateTempDirWithCallbackAndAssert(t)
+	defer createTempDirCallback()
+	assert.NoError(t, biutils.CopyDir(testDirPath, tempDirPath, true, nil))
+
+	testCases := []struct {
+		name                  string
+		sourceBranchResults   *results.SecurityCommandResults
+		expectedSbom          results.Sbom
+		expectedApiCallsCount map[string]int
+	}{
+		{
+			name:                "Source branch scan",
+			sourceBranchResults: nil,
+			expectedSbom: results.Sbom{
+				Components: []results.SbomEntry{
+					{
+						Component: "pip",
+						Version:   "25.0.1",
+						Type:      "Python",
+						XrayType:  "pypi",
+						Direct:    true,
+					},
+					{
+						Component: "pyyaml",
+						Version:   "5.2",
+						Type:      "Python",
+						XrayType:  "pypi",
+						Direct:    true,
+					},
+					{
+						Component: "werkzeug",
+						Version:   "1.0.1",
+						Type:      "Python",
+						XrayType:  "pypi",
+						Direct:    true,
+					},
+				},
+			},
+			expectedApiCallsCount: map[string]int{},
+		},
+		{
+			name: "Target branch scan",
+			sourceBranchResults: &results.SecurityCommandResults{
+				Targets: []*results.TargetResults{
+					{
+						ScanTarget: results.ScanTarget{
+							Target:     tempDirPath,
+							Name:       "",
+							Technology: "pip",
+						},
+						Sbom: results.Sbom{
+							Components: []results.SbomEntry{
+								{
+									Component: "pip",
+									Version:   "26.0",
+									Type:      "Python",
+									XrayType:  "pypi",
+								},
```

Comment:
I would change this to `werkzeug`, since pip is infered and at the CI job the version may be updated. this will cause this test to fail with no reason other than that the img that runs the job is updated